packages/experimental/ActivityIndicator/src/ActivityIndicator.types.tsx

```diff
@@ -0,0 +1,50 @@
+import { ColorValue, Animated, ViewProps } from 'react-native';
+import { SvgProps } from 'react-native-svg';
+
+export const activityIndicatorName = 'ActivityIndicator';
+/**
+ * Specifies the possible sizes of the ActivityIndicator
+ */
+export type ActivityIndicatorSize = 'xSmall' | 'small' | 'medium' | 'large' | 'xLarge';
+
+export interface ActivityIndicatorTokens {
+  /**
+   * ActivityIndicator element color
+   * @defaultValue 'BDBDBD' for light mode, '666666' for dark mode
+   */
+  activityIndicatorColor?: ColorValue;
```

Comment:
Forgot to mention this earlier. Because SVG's don't support ColorValue yet, I think the type here is better to be 'string'.

Reply:
Just curious, how do you know that svgs don't support ColorValue? And what's the difference?

Reply:
Nevermind, I think I got it.